project/PekkoDevelocityPlugin.scala

```diff
@@ -71,7 +76,6 @@ object PekkoDevelocityJdk9TestSettingsPlugin extends AutoPlugin {
   override lazy val trigger: PluginTrigger = allRequirements
   override lazy val requires: Plugins = DevelocityPlugin && Jdk9
 
-  // See https://docs.gradle.com/develocity/sbt-plugin/#capturing_test_data_in_a_custom_configuration
-  override lazy val projectSettings =
-    inConfig(Jdk9.TestJdk9)(DevelocityPlugin.testSettings)
-}
+  // See https://docs.gradle.com/develocity/sbt-plugin/#enabling_build_cache_in_a_custom_sbt_configuration
+  override lazy val projectSettings = DevelocityPlugin.develocitySettings(Jdk9.TestJdk9)
+}
```

Comment:
minor nit but could you add a new line at the end of the file?

Reply:
Sure, done